subsys/usb/device/Kconfig

```diff
@@ -72,6 +72,18 @@ config USB_DEVICE_BLUETOOTH_BIG_BUF
 	default y if BT_BUF_EVT_RX_SIZE > 125 # 2 byte header
 	default y if BT_BUF_CMD_TX_SIZE > 124 # 3 byte header
 
+config USB_EP0_MAX_PACKET_SIZE
+	int "Maximum size of endpoint 0 packet size"
+	default 64
+	help
+	  Maximum size of endpoint 0 packet size. This value goes to
+	  device descriptor and is used for EP0 configuration.
+	  It is wMaxPacketSize field (byte 7 of the device descriptor).
```

Comment:
Configuration Descriptor Endpoint Descriptors contain `wMaxPacketSize` (2 bytes hence the `w` prefix). What you mean is `bMaxPacketSize0` (1 byte hence the `b` prefix).

Reply:
This text `wMaxPacketSize field (byte 7 of the device descriptor)` was taken verbatim from USB2 specification section **5.5.3 Control Transfer Packet Size Constraints** where it is referred several times with **w** prefix.
But since it's incorrect I don't mind changing it here.